src/v/cluster/tx_registry_frontend.cc

```diff
@@ -316,7 +351,8 @@ ss::future<bool> tx_registry_frontend::try_create_tx_topic() {
     return _controller->get_topics_frontend()
       .local()
       .autocreate_topics(
-        {std::move(topic)}, config::shard_local_cfg().create_topic_timeout_ms())
+        {std::move(topic)},
+        config::shard_local_cfg().create_topic_timeout_ms() * partitions_amount)
```

Comment:
q: whats the rationale behind `* partitions_amount` part?

Reply:
`create_topic_timeout_ms` is too small to create 50 partitions in the ci environment. before it was use to create a single partition, `* partitions_amount` is the proportional increase. it's a bit meh but imho good enough, at least for the fix.